canonical/ubuntu.com
# Removed CUBE/edx code

## Done

- Removed all old references to CUBE (OpenEdx)

Fixes https://warthogs.atlassian.net/browse/CRED-229

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- check that no more CUBE/OpenEdx stuff is present

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/CRED-229

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
